src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -5990,31 +5990,30 @@ checkNullable:
         ''' of the parser.  If it is not available a diagnostic will be added to the returned value.
         ''' </summary>
         Private Function CheckFeatureAvailability(Of TNode As VisualBasicSyntaxNode)(feature As Feature, node As TNode) As TNode
-            Dim languageVersion = _scanner.Options.LanguageVersion
-            If CheckFeatureAvailability(languageVersion, feature) Then
+            Dim parseOptions = _scanner.Options
+            If CheckFeatureAvailability(parseOptions, feature) Then
                 Return node
             End If
 
             Dim featureName = ErrorFactory.ErrorInfo(feature.GetResourceId())
-            Return ReportSyntaxError(node, ERRID.ERR_LanguageVersion, languageVersion.GetErrorName(), featureName)
+            Return ReportSyntaxError(node, ERRID.ERR_LanguageVersion, parseOptions.LanguageVersion.GetErrorName(), featureName)
```

Comment:
Once we start adding features that are not enabled in any particular language version, this won't quite work. We can solve that problem at that time.